bundle/manifests/cluster-logging.clusterserviceversion.yaml

```diff
@@ -2208,7 +2208,7 @@ spec:
   provider:
     name: Red Hat
   relatedImages:
-  - image: quay.io/openshift-logging/vector:6.1
+  - image: quay.io/openshift-logging/vector:v0.43.1-rh
```

Comment:
We would actually push to 'v0.43.1' ref: https://github.com/openshift/release/pull/60752/files#diff-b40100f1ebd8bbcdef2aa6239c14de9c690b870b94c017b0cfd048517b7ba6e5R7

Reply:
fixed